tools/rsdl/dotnet/README.md

```diff
@@ -0,0 +1,9 @@
+# demo
+
+the demo program can be run like
+
+```sh
+dotnet run -- --input jetsons.rsdl --format JSON
```

Comment:
Would it be possible to:

- Add information how to compile program etc.
- rename output file to rapid etc.

Reply:
I got it running by following all hints in VS Code, e.g. the ones mentioned above.

Would be nice if someone could verify from scratch and complete the description.